README.md

```diff
@@ -139,7 +139,7 @@ Just try to follow these rules, so we can spend more time fixing bugs and implem
 The STL uses boost-math headers to provide P0226R1 Mathematical Special Functions. We recommend using [vcpkg][] to
 acquire this dependency.
 
-1. Install Visual Studio 2019 16.3 or later.
+1. Install Visual Studio 2019 16.4 or later.

```

Comment:
Should we change these occurrences to "Install the current preview release of Visual Studio"? Updating the versions is an annoyance, and "or later" seems pointless since this will forever be the most recent preview release.

Reply:
I think it should indicate the version we are testing with.

Reply:
Update two places to "16.4 preview 2", and changed the middle - step 7 below - to simply "Open Visual Studio".